README.md

```diff
@@ -18,46 +19,40 @@ with two packages: your project's package and the xtask package.
 ## Create a project using xtask
 
 * Create a new directory that will contains the two package of your project
-    and the workspace's `Cargo.toml`
+    and the workspace's `Cargo.toml`:
+
     ```console
     mkdir my-project
     cd my-project
     touch Cargo.toml
     ```
+
```

Comment:
Oh nice I love that you put spacing there so now I can read better in text format :hugs: 

I often feel that you and @o0Ignition0o are trying to make my life harder by making the text version of markdown harder to read.

Note that in this particular project we had a few issues where we reviewed the doc in the PR and didn't see any problem and only spotted them **after** the merge when reading the formatted readme on github. This is clearly a sign that we didn't polish the presentation of the text markdown (to be read in a text file)

Look for example how this (not markdown) text file is perfectly and easily readable without any rendering: https://github.com/rustminded/xtask-wasm/blob/main/LICENSE.Apache-2.0

This. Is. Beautiful.

Normally you're supposed to do the same in Markdown so people can read the doc with equal easiness in the rendered and the not rendered (text) version of the doc.

Here is a part of doc you wrote I find particularly bad because it is misaligned in the text version:

![image](https://user-images.githubusercontent.com/3018448/153177560-f38cae58-7bfb-4385-8a9e-8e918c81c881.png)

See how in this picture the first line of the paragraph is always offset on the left compared to the lines that come after. This is wrong imo. This is how it should look:

![image](https://user-images.githubusercontent.com/3018448/153177845-6985d606-0402-4913-a94c-7be260fb4e4a.png)

It's bit trickier with numbered list because when they exceed 1 digits, they become misaligned:

```
1. OK
2. OK
3. OK
9. OK
10. NOT OK
11. NOT OK
```

My way of fixing this is to use a fixed indent for both bullets and numbered list:
```
1.  OK
2.  OK
3.  OK
9.  OK
10. OK
11. OK

Next section:

 *  OK
 *  OK
 *  OK
 *  OK
```